public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
you can remove the manifest file because it contains default CRA JSON or update it according to you